roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/ruby
+username = "admin"
+password = "changeme"
```

Comment:
It may not be a safe assumption.  Here's why:
My user scenario is:
- clone the satellite (didnt pay lot of attention to the terminal instructions about capsule)
- change satellite's admin password as this is one of the very first things which admins (are supposed to) do
- Realize that capsules are disassociated, reviewing the logs, finding the command to reset capsule, run it - boom!